sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java

```diff
@@ -184,13 +184,13 @@ public void testSimpleCombineWithContext() {
   }
 
   @Test
-  @Category(ValidatesRunner.class)
+  @Category(NeedsRunner.class)
```

Comment:
For Combine there's an argument to bring some of them back since this is almost always implemented as a primitive, even though it is not one, technically. I would favor a JIRA TODO for factoring Combine into two suites - one for the composite's various configs but one that is focused on testing a runner's basic primitive implementation.

Reply:
Ok, I've reverted these changes and instead subdivided the tests into subclasses.